src/components/MessageBar/MessageBar.scss

```diff
@@ -149,12 +132,10 @@ $MessageBar-padding: 8px;
 }
 
 .ms-MessageBar-dismissal {
-  padding: 8px;
   @include focus-border();
 }
-
-.ms-MessageBar-link {
-  @include padding-left(4px);
+.ms-MessageBar-actions-oneline .ms-MessageBar-dismissal {
```

Comment:
I know this is being ported over, but we would typically use camelCase for classes like this instead of dashes. So this should be `ms-MessageBar-actionsOneline`.